org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -6184,6 +6184,7 @@ class SemanticTokens {
 	List<Integer> data
 
 	new() {
+		this.data = java.util.Collections.emptyList();
```

Comment:
You could use import here

Reply:
```suggestion
		this.data = new ArrayList()
```
as in other similar cases in `Protocol.xtend`.

`Collections.emptyList()` cannot be used in this case, as it returns an immutable list.